### PR TITLE
Auto/steering wheel status

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Description

add wheel status as iv message

## Review Procedure

lauch pacmod interface

## Remarks

this PR should be merged before
https://github.com/tier4/AutowareArchitectureProposal_msgs/pull/13

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by taikitanaka3** [comment]
@tkimura4 
can you take a look at?

File: vehicle/pacmod_interface/src/pacmod_interface/pacmod_interface.cpp [hunk 1]
```diff
@@ -240,6 +242,14 @@ void PacmodInterface::callbackPacmodRpt(
   header.frame_id = base_frame_id_;
   header.stamp = get_clock()->now();
 
+  /* publish steering wheel status */
+  {
+    SteeringWheelStatusStamped steering_wheel_status_msg;
+    steering_wheel_status_msg.stamp = header.stamp;
+    steering_wheel_status_msg.data = steer_wheel_rpt_ptr_->output;
```
Comment (tkimura4) [inline vehicle/pacmod_interface/src/pacmod_interface/pacmod_interface.cpp:249]: current_steer_wheel

Comment (tkimura4) [reply vehicle/pacmod_interface/src/pacmod_interface/pacmod_interface.cpp:249]: (steering_offsetは考慮しない？)

**Review by tkimura4 — APPROVED** [review]

File: vehicle/pacmod_interface/src/pacmod_interface/pacmod_interface.cpp (hunk 1 above)
Comment (taikitanaka3) [reply vehicle/pacmod_interface/src/pacmod_interface/pacmod_interface.cpp]: Memo ステアオフセットのズレをみたいので一旦Offsetは考慮しない値を出力。
3731643

**Pull request merged**